packages/api-client-core/package.json

```diff
@@ -24,7 +24,7 @@
     "@urql/exchange-multipart-fetch": "^1.0.0",
     "cross-fetch": "^3.0.6",
     "gql-query-builder": "^3.6.0",
-    "graphql": "^15.5.0",
+    "graphql": "^16.5.0",
```

Comment:
We should be using the newer version, urql is compatible with it and it has some bug fixes. I changed it because it aligns with other stuff so we don't have type clashes.